src/tests/flow/CMakeLists.txt

```diff
@@ -50,31 +50,32 @@
 ################################
 # Flow Unit Tests
 ################################
-
-set(FLOW_TESTS  t_flow_data
-                t_flow_timer
-                t_flow_registry
-                t_flow_workspace
-                t_flow_workspace_adv_manage)
-
-if(PYTHON_FOUND)
-    list(APPEND FLOW_TESTS t_flow_python_interpreter
-                           t_flow_python_script_filter)
-endif()
-
-################################
-# Add tests
-################################
-
-#(tests depend on ascent b/c we are using some of its utils in the tests)
-
-message(STATUS "Adding flow lib unit tests")
-foreach(TEST ${FLOW_TESTS})
-    add_cpp_test(TEST ${TEST} DEPENDS_ON ascent_flow ascent)
-endforeach()
-
-if(PYTHON_FOUND AND ENABLE_PYTHON)
-    add_subdirectory("python")
-else()
-    message(STATUS "Python disabled: Skipping ascent python module tests")
+if (ENABLE_SERIAL)
+    set(FLOW_TESTS  t_flow_data
```

Comment:
In theory these don't need to depend on ENABLE_SERIAL, flow is always a serial lib & is built for use in both non-mpi and mpi-cases, it does not link to any VTK-h or VTK-m stuff. 

I do see we have ascent on as a dep, I think that is a mistake -- we should remove it as a dep.

Reply:
Yes, I was able to do that.